proxy/envoy/testdata/envoy-cb-policy.json.golden

```diff
@@ -311,74 +214,22 @@
         ]
       },
       {
-        "name": "outbound:hello.default.svc.cluster.local:http-status",
-        "service_name": "hello.default.svc.cluster.local:http-status",
-        "connect_timeout_ms": 1000,
-        "type": "sds",
-        "lb_type": "round_robin"
-      },
-      {
         "name": "outbound:world.default.svc.cluster.local:custom",
         "service_name": "world.default.svc.cluster.local:custom",
         "connect_timeout_ms": 1000,
         "type": "sds",
-        "lb_type": "round_robin",
-        "max_requests_per_connection": 100,
-        "circuit_breakers": {
-          "default": {
-            "max_connections": 100,
-            "max_pending_requests": 100,
-            "max_requests": 100
-          }
-        },
-        "outlier_detection": {
-          "consecutive_5xx": 10,
-          "interval_ms": 30000,
-          "base_ejection_time_ms": 15500,
-          "max_ejection_percent": 100
-        }
-      },
-      {
-        "name": "outbound:world.default.svc.cluster.local:http",
-        "service_name": "world.default.svc.cluster.local:http",
-        "connect_timeout_ms": 1000,
-        "type": "sds",
-        "lb_type": "round_robin",
-        "max_requests_per_connection": 100,
-        "circuit_breakers": {
-          "default": {
-            "max_connections": 100,
-            "max_pending_requests": 100,
-            "max_requests": 100
-          }
-        },
-        "outlier_detection": {
-          "consecutive_5xx": 10,
-          "interval_ms": 30000,
-          "base_ejection_time_ms": 15500,
-          "max_ejection_percent": 100
-        }
+        "lb_type": "round_robin"
```

Comment:
These are now part of CDS output. I need to add a unit test with the sample output from CDS.